docs/Deploy.md

```diff
@@ -0,0 +1,136 @@
+# Contract Deployment Checklist
+
+The following content will guide you through deploying and configuring all smart contracts to a new network (or to deploy a new staging environment).
+
+If you only want to deploy a new diamond contract and use existing deployed facets, please skip to [this section](#deploy-new)
+
+If you want to deploy an immutable diamond, please skip to this section
+
+## Prerequisites
+
+1. Prepare your .env file to have the following values set:
+
+**CREATE3_FACTORY_ADDRESS**= 0x93FEC2C00BfE902F733B57c5a6CeeD7CD1384AE1
+(can be verified here: [https://github.com/lifinance/create3-factory/tree/main/deployments](https://github.com/lifinance/create3-factory/tree/main/deployments))
+
+**ETH_NODE_URI_\<NETWORK\>**=<add your own RPC link here> (NETWORK being the network you deploy to e.g. MAINNET, POLYGON or BSC)
+(e.g. [https://1rpc.io/eth](https://1rpc.io/eth))
+
+**\<NETWORK\>_ETHERSCAN_API_KEY**=<add your own Etherscan API key here>
+
+> > if you deploy to another network, make sure you have both variables set for that specific network. E.g.: **ETH_NODE_URI_GOERLI** and **GOERLI_ETHERSCAN_API_KEY** for deployments to Goerli etc.
+
+**PRODUCTION**=false (set to **true** if you want to deploy to production obviously, but confirm with @Edmund Zynda beforehand)
+
+2. Install required packages
+
+[https://github.com/foundry-rs/foundry](https://github.com/foundry-rs/foundry)
+
+[https://github.com/charmbracelet/gum](https://github.com/charmbracelet/gum)
+
+[https://github.com/stedolan/jq](https://github.com/stedolan/jq)
+
+3. Merge your branch with latest master to make sure you have all the latest addresses stored in your deployments folder
+
+Scripts:
+
+- /scripts/deploy.sh (for deploying any of our contracts)
+  - execute with command `./scripts/deploy.sh` in your console
+- /scripts/diamond-update.sh (update / add to diamond)
+  - execute with command `./scripts/diamond-update.sh` in your console
+- /scripts/update-periphery.sh (add new contract to periphery registry)
+  - execute with command `./scripts/update-periphery.sh` in your console
+
+## Deploy facet and LiFiDiamond contracts
+
```

Comment:
Perhaps add that the CREATE3_FACTORY has to be deployed to that chain first using the other repository.